CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# the global owners will be requested for
+# review when someone opens a pull request.
+*       @istio/reviewers
+
+# Per component owners. Prefix folders with / so that the owners
+# will be requested for reviews for all subdirectories under that
+# folder in the root
+/.circleci/ @rshriram @costinm # please do not change this
```

Comment:
Someone from test & release group here?

Reply:
Sure, anyone who is is committed to maintain CI can get on this list - and we should have a similar one for the release/ directory.

Reply:
In due time, when both of us are confident that they know all the ins and outs of circleci.

Reply:
I was about to put Matt and someone for the release directory. But the point here is that pkg/ and tests/ are collective responsibility of everyone in the reviewers team.

Reply:
The only way for them to get confident is by actively contributing :)